frontend_lib/src/component/Timeline/CommentAsFile.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import Avatar, { AVATAR_SIZE } from '../Avatar/Avatar.jsx'
+import {
+  buildFilePreviewUrl,
+  formatAbsoluteDate,
+  removeExtensionOfFilename
+} from '../../helper.js'
+import EmojiReactions from '../../container/EmojiReactions.jsx'
+
+export const CommentAsFile = props => {
```

Comment:
`TimelineFilePreview`?
Even though this component could be used outside a timeline…

Reply:
fixed

Reply:
renamed to CommentFilePreview